snaps_boot/provision/ipmi_utils.py

```diff
@@ -35,9 +41,16 @@ def reboot_disk(boot_conf):
 def __reboot_all(boot_conf, boot_order):
     creds = get_ipmi_creds(boot_conf)
     for ip, user, password in creds:
-        command = Command(ip, user, password)
-        __set_boot_order(command, boot_order)
-        __reboot(command)
+        try:
+            command = Command(ip, user, password)
+            __set_boot_order(command, boot_order)
+            __reboot(command)
+        except IpmiException as e:
+            # print the error message and server bmc information
+            # will continue to reboot the rest of servers
+            logger.error(e)
+            logger.warn('Failed to reboot server with BMC IP: %s, user: %s', ip, user)
+            logger.debug('BMC password: %s', password)
```

Comment:
Add error here (or in the warn if you want it more visible). You can just add another %s in the string and append the 'e' variable to the function parameters

Reply:
I like this feedback. Fixed it in the latest patch.